coding-concepts.js

```diff
@@ -18,38 +18,40 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: Console log prints the requested code. The print that was requested was our declared variable cohort's length of data.
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "o"
+// a) Correct answer : o
+// b) Verify and explain: The print that was requested was the data that was in the 4th indices of our declared varible greeting. I incorrectly put "o" thinking that it would output a string version.
```

Comment:
Technically it did output a string. The terminal may not note it but you are correct that it is a string.